projects/go-lib/src/lib/components/go-side-nav/go-side-nav/go-side-nav.component.scss

```diff
@@ -28,3 +27,39 @@
     width: $side-nav-width--collapsed;
   }
 }
+
+.go-side-nav__nav-items {
+  overflow-y: auto;
+}
+
+.go-side-nav__app {
+  margin-top: auto;
+  background: $theme-dark-bg-hover;
+  color: $theme-dark-color;
+  height: 2.75rem;
+  padding: .5rem .625rem;
+  font-size: 1.25rem;
+  display: flex;
+  align-items: center;
+}
+
+.go-side-nav__app-icon {
+  display: flex;
+  padding: 0 1rem 0 .5rem;
+}
+
+.go-side-nav__app-drawer {
+  cursor: pointer;
+  margin-left: auto;
+
+  ::ng-deep .go-side-nav__app-drawer--icon {
```

Comment:
Don't love this.. but: shrug:

Reply:
yeah, I didn't know a better way of handling this 😞